platform/solr/solr-dependencies/pom.xml

```diff
@@ -27,7 +27,6 @@
         <info.ganglia.gmetric4j.version>1.0.7</info.ganglia.gmetric4j.version>
         <org.slf4j.version>1.7.7</org.slf4j.version>
         <log4j.version>1.2.17</log4j.version>
-        <jts.version>1.13</jts.version>
     </properties>
     <dependencies>
```

Comment:
Updated: I went through and pulled in the solr 6.6 webapps script files.  Also updated all dependencies into the dependencies jar.  Built and confirmed the right dependencies were being added (by expanding the dependencies jar as well building on an empty m2 and confirming the right versions were pulled in.